index.html

```diff
@@ -8,22 +8,146 @@
+      </div>
+      </div>
+    </div>
+    <div id="project-section3" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    <div id="project-section5" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    <div  id="project-section7" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    <div id="project-section9" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""> </button>
+      </div>
+    </div>
+    <div id="project-section11" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    </div>
+  </section>
+  <section id="about-section">
+  <div id="about-sction-div">
+    <h2>About</h2>
+    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. 
+       Labore praesentium, ipsa necessitatibus maioresquaeiusto eveniet
+       asperiores at alias debitis deserunt excepturi, esse sequi.</p>
+    <button type="button">Get my cv <img src="./images/arrow-down.png" alt=""></button>
```

Comment:
- [ ] Your buttons are supposed to be a link as well. try to add an anchor tag so that your buttons are links too. The design is supposed to stay similar to the one from provided figama.

Reply:
Done